rand_distr/tests/value_stability.rs

```diff
@@ -112,8 +132,16 @@ fn unit_circle_stability() {
 #[test]
 fn unit_sphere_stability() {
     test_samples(2, UnitSphere, &[
-        [0.03247542860231647f64, -0.7830477442152738, 0.6211131755296027],
-        [-0.09978440840914075, 0.9706650829833128, -0.21875184231323952],
+        [
+            0.03247542860231647f64,
+            -0.7830477442152738,
+            0.6211131755296027,
+        ],
+        [
+            -0.09978440840914075,
+            0.9706650829833128,
+            -0.21875184231323952,
+        ],
         [0.2735582468624679, 0.9435374242279655, -0.1868234852870203],
```

Comment:
This is where it does get ugly... should we just skip formatting of all these values? There are a lot of them... probably still better however.

Reply:
Apart from inconsistency with the third value, I think such formatting is fine. Using `rustfmt::skip` on each test would be too noisy, while disabling it for the whole module would probably be an overkill.

Reply:
No, don't disable for the whole module. I still prefer the old formatting, but `rustfmt`'s output is acceptable.